src/main/java/org/openmrs/addonindex/legacy/LegacyController.java

```diff
@@ -75,13 +75,14 @@ public String checkUpdate(@PathVariable("moduleId") String moduleId) throws Exce
 	@RequestMapping(method = RequestMethod.GET, value = "findModules")
 	public LegacyFindModulesResponse findModules(
 			@RequestParam(value = "sSearch", required = false) String query,
+			@RequestParam(value = "tags", required = false) String tag,
```

Comment:
If there's someplace that's testing the search of `type` and `query` in any way, then you should add an equivalent search for tag. Otherwise, ignore it. (This will be the only time I tell you it's okay to ignore writing a test. But there's no worthwhile way of testing this.)